transport/http/transport.go

```diff
@@ -21,11 +21,14 @@
 package http
 
 import (
+	"context"
 	"net"
 	"net/http"
 	"sync"
 	"time"
 
+	"golang.org/x/net/proxy"
```

Comment:
You’re both correct that the order is (native, internal, external). "golang.org/x/net/proxy" is external so should go down with opentracing. If there’s a strong argument that it’s native, the space above the import should be below.